bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableScanConfiguration.java

```diff
@@ -128,19 +139,16 @@ public Builder withRequest(ValueProvider<ReadRowsRequest> request) {
      * @return The {@link CloudBigtableScanConfiguration.Builder} for chaining convenience.
      */
     Builder withKeys(byte[] startKey, byte[] stopKey) {
-      Preconditions.checkNotNull(request, "Request cannot be empty.");
-      Preconditions.checkState(request.isAccessible(), "Request must be accessible.");
-      final ByteString start = ByteStringer.wrap(startKey);
-      final ByteString stop = ByteStringer.wrap(stopKey);
-      return withRequest(
-          request
-              .get()
-              .toBuilder()
-              .setRows(
-                  RowSet.newBuilder()
-                      .addRowRanges(
-                          RowRange.newBuilder().setStartKeyClosed(start).setEndKeyOpen(stop)))
-              .build());
+      Preconditions.checkNotNull(scan, "Request cannot be empty.");
```

Comment:
I think it makes sense to say request here, but please look at any comments or errors that should say scan instead of request.